gridy/Sources/Presentations/PlanBoard/TimelineLayout/ListArea/LargeTaskElementView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  LargeTaskElementView.swift
+//  gridy
+//
+//  Created by xnoag on 10/9/23.
+//
+
+import SwiftUI
+
+struct LargeTaskElementView: View {
+    @State private var isTaskElementHovering = false
+    @State private var isLeftButtonHovering = false
+    @State private var isRightButtonHovering = false
+    @Binding var isLeftButtonClicked: Bool
+    @Binding var isRightButtonClicked: Bool
+    @FocusState private var isTextFieldFocused: Bool
+    @Binding var largeTaskElementTextField: String
+    @State private var isEditing = false
+    
+    var body: some View {
+        Rectangle()
+            .frame(minWidth: 266, idealWidth: 266, maxWidth: 532)
+            .frame(height: 48)
+            .foregroundStyle(.white)
+            .border(.gray, width: 0.5)
+            .overlay {
+                if !isEditing {
+                    Text(largeTaskElementTextField)
+                        .lineLimit(2)
+                        .foregroundStyle(.black)
+                        .font(.custom("Pretendard-Regular", size: 16))
+                        .padding(.horizontal, 8)
+                }
+            }
+            .overlay {
+                if isTaskElementHovering && largeTaskElementTextField.isEmpty {
+                    Button(action: {
+                        isEditing = true
+                        isTaskElementHovering = false
+                        isTextFieldFocused = true
+                    }) {
+                        Rectangle()
+                            .foregroundStyle(.gray.opacity(0.2))
+                            .cornerRadius(6)
+                            .overlay {
+                                Text("New Task")
+                                    .foregroundStyle(.gray)
+                                    .font(.custom("Pretendard-Regular", size: 16))
+                            }
+                            .padding(2)
+                    }
+                    .buttonStyle(PlainButtonStyle())
```

Comment:
아래처럼 괄호를 최소화해서 가독성을 높일 수 있도록 작성하거나,
```swift
Button {
  // action
} label : {
  // label 
}
```
action에 대한 클로저를 아래 코드처럼 따로 분리하고 호출해서 사용해서 가독성을 높이는 방법도 있어요!
버튼에 대한 동작을 따로 작성하기 때문에 나중에 동작이 추가되더라도 조금 더 독립적으로 유지보수가 가능해질 것 같아요!!
```suggestion
                    Button(action: namingPlz) {
                            Rectangle()
                                .foregroundStyle(.white.opacity(0.1))
                                .frame(minWidth: 266-48, idealWidth: 266-48, maxWidth: 532-48)
                                .frame(height: 48)
                        }
                        .buttonStyle(PlainButtonStyle())
                        // ... 중략 ...
                }
        }
        
        private func namingPlz() {
                isEditing = true
                isTextFieldFocused = true
                isTaskElementHovering = false
        }
}
```

Reply:
[fix][#19] 버튼 간소화